app/controllers/works_controller.rb

```diff
@@ -6,12 +6,11 @@ class WorksController < ApplicationController
   before_action :check_deposit_job_started, only: %i[show edit]
   before_action :set_work_form_from_cocina, only: %i[show edit]
   before_action :set_content, only: %i[show edit]
-  before_action :set_status, only: %i[show edit]
+  before_action :set_status, only: %i[edit]
 
   def show
     authorize! @work
-    @status_presenter = StatusPresenter.new(status: @status)
-    @purl_link_presenter = PurlLinkPresenter.new(druid: @work.druid)
+    @work_presenter = WorkPresenter.new(work: @work, work_form: @work_form)
```

Comment:
Now the entire works show page is driven by a single presenter. This is how we use presenters in Argo, IIRC.